server/externs/cookieParser.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @fileoverview Node.js cookieParser externs.
+ */
+
+/** @const */
```

Comment:
I didn't notice this but this seems to be a constructor based on the way you use it.